citesphere/src/main/java/edu/asu/diging/citesphere/web/user/dto/MovedItemsData.java

```diff
@@ -0,0 +1,16 @@
+package edu.asu.diging.citesphere.web.user.dto;
+
+import java.util.List;
+
+public class MovedItemsData {
```

Comment:
this hold items to be moved, not moved, right?

Reply:
When items are moved to a collection, ajax call sends connection key and items key list as a JSON string. This class is used in `MoveItemController` to parse JSON string parameter back to an object.

Reply:
Right, so it is for the items to be moved, not moved, right? `MovedItemsData` sounds like the items have already been moved, but they are going to be moved, right?

Reply:
Yes, the items that are going to be moved. Do you recommend changing name?

Reply:
I changed name of this class to `MoveItemsRequest`